Mappy.Tests/SortedSetTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Mappy.Tests
+{
+    public class SortedSetTests
+    {        
+        class Customer
+        {
+            public int CustomerId { get; set; }
+            public string FirstName { get; set; }
+            public string LastName { get; set; }
+            public SortedSet<Order> Orders { get; set; }
+        }
+
+        class Order
+        {
+            public int OrderId { get; set; }
+            public decimal OrderTotal { get; set; }
+        }
+
+        enum UserSkill
+        {
+            SkillA = 1,
+            SkillB = 2,
+            SkillC = 3
+        }
+
+        class User
```

Comment:
Looks like User and UserSkill is not used

Reply:
You are right!